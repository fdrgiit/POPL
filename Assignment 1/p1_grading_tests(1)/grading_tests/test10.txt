
      xyz.1234int

   	 	 abcd456xyz.0real          

             xyz.1234INT               				   

        abcd456xyz.0REAL      				


