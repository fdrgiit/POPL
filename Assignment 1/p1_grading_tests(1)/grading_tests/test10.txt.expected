2 37 xyz 
2 33 .
2 34 1234 
2 37 int 
4 37 abcd456xyz 
4 33 .
4 34 0 
4 37 real 
6 35 xyz.1234INT 
8 36 abcd456xyz.0REAL 
